src/api/Models/UserWorkgroupRole.cs

```diff
@@ -20,22 +20,12 @@ public class UserWorkgroupRole
     [Column("id_wgp_fk")]
     public int WorkgroupId { get; set; }
 
-    /// <summary>
-    /// Gets or sets the <see cref="User"/>.
-    /// </summary>
-    public User User { get; set; }
-
-    /// <summary>
-    /// Gets or sets the <see cref="Workgroup"/>.
-    /// </summary>
-    public Workgroup Workgroup { get; set; }
-
```

Comment:
FYI: Diese Navigation-Properties führten zu zirkulären Referenzen beim Laden der User-Liste in der Applikation. Da diese Properties ohnehin nicht verwendet werden, habe ich sie gelöscht. Das Problem mit den zirkulären Referenzen kann man angehen, sobald man die Navigation-Properties wirklich braucht.